CeeFM/Decideify
# Post Reactions V3 Now Live!

**Description**
Post reactions functionality and styling has been updated! One reaction (tim robinson awkwardly looking around) has been removed to make the max number of reactions an even and predictable number for styling purposes. When no reactions are added to a post, there's an H4 now that takes up that space and encourages users to add one. Adding the first post reaction will have it render in place of the H4. Removing the only post reaction will remove all post reaction buttons and replace it with the H4. Any post with more than 6 unique post reactions will render the remaining post reactions on a second row. If there's only one row of post reactions, some padding will be applied to the top to make the single row appear vertically centered.

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- On social.js or the /social page:
  - When no reactions are added to a post, there's an H4 now that takes up that space and encourages users to add one. 
  - Adding the first post reaction will have it render in place of the H4. 
  - Removing the only post reaction will remove all post reaction buttons and replace it with the H4. 
  - Any post with more than 6 unique post reactions will render the remaining post reactions on a second row. 
  - If there's only one row of post reactions, some padding will be applied to the top to make the single row appear vertically centered.

Checklist
